### ISSUE TITLE
Send appealed posts back to the mod queue

### ISSUE THREAD

**Issue by nonamethanks** [danbooru/danbooru#4568]
Currently appeals are rarely used. There's many reasons for this, including lack of accessibility for the appeal queue, the fact that you can only appeal once a day, and the fact that forums are easier to use. Also, appeals are not as effective as flags: they are passive rather than active. A flag requires someone to save a post, while an appeal will just go ignored if nobody is checking their page.

Sending them back to the queue would be a bit more effective as it would expose the posts again to the eyes of approvers.
